src/test/security/test_acm.py

```diff
@@ -0,0 +1,73 @@
+from mock import MagicMock
+import unittest
+
+from spacel.aws import ClientCache
+from spacel.security.acm import AcmCertificates
+from test import REGION
+
+TEST_EXAMPLE_COM = '111111'
+STAR_EXAMPLE_COM = '222222'
+STAR_TEST_DOUBLE_COM = '333333'
+STAR_STAR_DOUBLE_COM = '444444'
+
+CERTIFICATE_LIST = [
+    {'DomainName': 'test.example.com', 'CertificateArn': TEST_EXAMPLE_COM},
+    {'DomainName': '*.example.com', 'CertificateArn': STAR_EXAMPLE_COM},
+    {'DomainName': '*.test.double.com', 'CertificateArn': STAR_TEST_DOUBLE_COM},
+    {'DomainName': '*.*.double.com', 'CertificateArn': STAR_STAR_DOUBLE_COM}
```

Comment:
but far as I can see it's only used to check if we use `STAR_TEST_DOUBLE_COM` instead, so.. :+1: